html/index.html

```diff
@@ -103,7 +103,7 @@ <h1 class="align-text-center">
                         href="https://www.youtube.com/playlist?list=PLjA0bhxgryJ2Ts4GJMDA-tPzVWEncv4pb"
                         class="nav-link">Videos</a></li>
                 <li class="nav-item"><a target="_blank"
-                        href="https://www.networktocode.com/academy/enterprise/nautobot-package-iv/"
+                        href="https://networktocode.com/how-we-do-it/enablement-adoption/network-automation-academy/"
```

Comment:
Should this be the same link as we have at line 320 for the footer?

Reply:
Yes, that is correct. Thanks for the catch. They should both be changed now.